mtp_noms_ops/templates/settings/job-information.html

```diff
@@ -0,0 +1,97 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load mtp_common %}
+
+{% block page_title %}
+    {{ view.title }}
+    –
+    {{ block.super }}
+{% endblock %}
+
+{% block inner_content %}
+
+    <header>
+        <h1 class="heading-xlarge">{{ view.title }}</h1>
+    </header>
+
+    {% include 'mtp_common/includes/message_box.html' %}
+
+    <div class="grid-row">
+        <div class="column-two-thirds">
+
+            <form method="post" class="mtp-job-information-form">
+                {% csrf_token %}
+
+                {% if form.errors %}
+                {% include 'mtp_common/forms/error-summary.html' with form=form only %}
```

Comment:
personally, i prefer indenting django template tags in the same way as html elements so that you can see where the nesting happens